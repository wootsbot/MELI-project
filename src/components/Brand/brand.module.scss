@import 'styles/helpers';

.wrapper {
  height: pixels-to-rem(35);
}

.brand {
  cursor: pointer;
  height: 100%;
}
