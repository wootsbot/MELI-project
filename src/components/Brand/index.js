/* eslint-disable @next/next/no-img-element */
/* eslint-disable @next/next/link-passhref */
import Link from 'next/link';

import styles from './brand.module.scss';

function Brand() {
  return (
    <div className={styles.wrapper}>
      <Link href="/">
        <img className={styles.brand} src="/static/images/png/Logo_ML@2x.png" alt="search" />
      </Link>
    </div>
  );
}

export default Brand;
