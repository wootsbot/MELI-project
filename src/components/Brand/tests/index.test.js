import { render } from '@testing-library/react';

import Brand from '..';

describe('<Brand />', () => {
  it('Should render correctly', async () => {
    const { container } = await render(<Brand />);

    expect(container).toMatchSnapshot();
  });
});
