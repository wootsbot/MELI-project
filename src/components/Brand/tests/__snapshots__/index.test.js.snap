// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Brand /> Should render correctly 1`] = `
<div>
  <div
    class="wrapper"
  >
    <img
      alt="search"
      class="brand"
      src="/static/images/png/Logo_ML@2x.png"
      tabindex="1"
    />
  </div>
</div>
`;
